kornia/contrib/sam/prompter.py

```diff
@@ -33,15 +43,22 @@ def __post_init__(self) -> None:
 
     @property
     def keypoints(self) -> Tensor | None:
+        """The keypoints from the `points`"""
         return self.points[0] if isinstance(self.points, tuple) else None
 
     @property
     def keypoints_labels(self) -> Tensor | None:
+        """The keypoints labels from the `points`"""
         return self.points[1] if isinstance(self.points, tuple) else None
 
 
 class ImagePrompter:
```

Comment:
This to me should be the one sunclassing BaseModel (or Algorithm)

Reply:
Hmm, for me, this is a BaseClass for models that will need Prompts

The BaseModel (or Algorithm) I think is more aligned to the Models itself